lib/Sema/TypeCheckConcurrency.cpp

```diff
@@ -1786,7 +2008,12 @@ namespace {
             ConcurrentReferenceKind::CrossActor);
       }
 
+      bool isDistributedActor = false;
```

Comment:
This seems to only be used to be passed down to `noteIsolatedActorMember`.

Reply:
and now it's gone -- the `noteIsolated` detects it automatically 👍